tests/defences/test_spatial_smoothing.py

```diff
@@ -39,6 +39,7 @@ def test_ones(self):
 
         # Start to test
         for window_size in range(1, 20):
+            print('Window size: {}'.format(window_size))
```

Comment:
Why output the same info in logger and print?

Reply:
I agree, the print statement needs to be removed again, it got forgotten. I had added it for debugging because this test sometimes fails and the test logging for this test is not showing up in the Travis logs.

Reply:
Fixed in 614a65395b392fe959e50c267e6cc6517e4d22f5